app/src/main/res/xml/private_browsing_preferences.xml

```diff
@@ -2,20 +2,16 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
     <androidx.preference.Preference
```

Comment:
I noticed that you remove the namespace in some of the other PRs, can it be removed here? (not needed)

Reply:
Android Studio warns me on not used ones, then I'm removing them. `xmlns:android` is used in there.

Reply:
Sorry, I meant the usage here:
```suggestion
    <Preference
```
Can it get removed?

Reply:
Ahhh yes we can :)